src/util/BaseLoginController.js

```diff
@@ -92,8 +92,14 @@ define(['okta', 'vendor/lib/q'], function (Okta, Q) {
     toggleButtonState: function (state) {
       var button = this.$el.find('.button');
       button.toggleClass('link-button-disabled', state);
-    }
+    },
 
+    // Event triggered at after a page is rendered along with the classname to identify the page
+    postRender: function() {
+      _.defer(_.bind(function () {
+        this.trigger('pageRenderedEvent',[this.className]);
```

Comment:
Hmm, also, are we sure this is the right time to do this? I think this won't actually be consistent:
- we have animations, so at the time we're rendered/postRendered it's possible to not have added the controller to the dom yet (it won't be a problem on primaryAuth, but could be a problem on the other pages that go through the animation).
- Check out BaseLoginRouter.render